code/game/objects/items/devices/minedetector.dm

```diff
@@ -0,0 +1,102 @@
+/obj/item/device/mine_detector
+	name = "mine detector"
+	desc = "A device capable of detecting mines and traps in a range."
+
+	///The types that are detected by this device
+	var/list/detectable_types = list(/obj/item/landmine, /obj/item/storage/box/traps)
```

Comment:
why is this detecting boxes of traps, instead of actual traps?
unless that is intentional? idk

Reply:
Because i'm stupid and can't read, i will fix it